guillotina/db/storages/pg.py

```diff
@@ -782,21 +782,21 @@ def objects_table_name(self):
 
     @restart_conn_on_exception
     async def open(self):
-        conn = await self.pool.acquire(timeout=self._conn_acquire_timeout)
+        conn = await self.pool.acquire()
```

Comment:
Are we sure we don't want to timeout here?

Otherwise, this could hang indefinitely.

Reply:
Any idea?

Reply:
Doing it `conn = await self.pool.acquire(timeout=self._conn_acquire_timeout)` isn't safe because it's doing a wait for and it can get cancelled? Is that the problem?

Reply:
I hope this clarifies the race condition:

![f3187af5_IMG_20191011_184617](https://user-images.githubusercontent.com/775189/66669379-a5d4b000-ec57-11e9-8422-4a103a529ea8.jpg)